hack/generated/pkg/armclient/raw_client.go

```diff
@@ -23,7 +23,7 @@ type Client struct {
 	Host string
 }
 
-const UserAgent = "k8sinfra-generated"
+const UserAgent = "azure-service-controller-generated"
```

Comment:
It's `aso-controller` or something in the actually exename? Wondering if we should use that? This reads sorta weird?

Reply:
We should also have the version in here. Will open an issue for that.